helix-term/src/commands.rs

```diff
@@ -2561,14 +2566,18 @@ fn append_mode(cx: &mut Context) {
     doc.set_selection(view.id, selection);
 }
 
+fn file_picker_in(cx: &mut Context, root: PathBuf) {
+    let picker = ui::file_picker(root, &cx.editor.config());
+    cx.push_layer(Box::new(overlaid(picker)));
+}
+
```

Comment:
adding a helper for two lines of code is really nod necessary and will cause merge conflicts with other changes to the picker